draft-ietf-rtcweb-jsep.xml

```diff
@@ -3619,10 +3619,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
                     <t>Or, if the ICE ufrag and password values have
                     changed, trigger the ICE Agent to start an ICE
-                    restart and begin gathering new candidates for the
-                    media section, as defined in <xref target="RFC5245"
-                    />, Section 9.1.1.1, unless it has been marked as
-                    bundle-only.</t>
+                    restart and, unless it has been marked as
```

Comment:
My original comment was unclear - I was trying to suggest that we probably want to skip any ICE restart processing (either offer or answer) if this section is bundle-only.

e.g. "Or, if the ICE ufrag and password values have changed, and it has not been marked as bundle-only, begin gathering..."

May also want to say "also start checks on that media section" to make it clear that starting checks is in addition to gathering.